src/ensembl/src/content/app/entity-viewer/services/entity-viewer-storage-service.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { EntityViewerGeneralState } from 'src/content/app/entity-viewer/state/general/entityViewerGeneralState';
+import storageService, {
+  StorageServiceInterface,
+  StorageType
+} from 'src/services/storage-service';
+
+export enum StorageKeys {
+  GENERAL_STATE = 'entityViewer.generalState',
+  SIDEBAR_STATE = 'entityViewer.sidebarState',
+  GENE_VIEW_STATE = 'entityViewer.geneViewState'
```

Comment:
`SIDEBAR_STATE` and `GENE_VIEW_STATE` don't seem to be used anywhere?

Reply:
Removed them